gradle.properties

```diff
@@ -1,4 +1,4 @@
-version=1.0.0-dev.31.6
+version=1.0.0-dev.31.7
```

Comment:
Probably a left over from local testing?

Reply:
Did it on purpose, thought that it's a good idea to bump version
Gonna remove it though to stick with the default release process